spec/support/authentication_steps.rb

```diff
@@ -0,0 +1,9 @@
+module AuthenticationSteps
+  def and_i_sign_in(user)
+    sign_in_page = SignInPage.new
+    sign_in_page.load
+    sign_in_page.email_field.set user.email
+    sign_in_page.password_field.set 'Password1!'
```

Comment:
Apart from if the test data changes it would have to be changed twice? Fair enough though.